java/tools/pom.xml

```diff
@@ -57,10 +57,12 @@
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-core</artifactId>
+      <scope>test</scope>
```

Comment:
@lidavidm I guess this is too bad, this is related to the issue raised by @danepitkin as well. 

https://github.com/apache/arrow/pull/40149#issuecomment-1958526213 

Should we rather do this?

Reply:
Can we just drop the Jackson dependency? What are we using from it?

Reply:
```
import com.fasterxml.jackson.core.util.DefaultPrettyPrinter;
import com.fasterxml.jackson.core.util.DefaultPrettyPrinter.NopIndenter;
import com.fasterxml.jackson.databind.ObjectMapper;
import com.fasterxml.jackson.databind.SerializationFeature;
```

These are the classes we are using.

Reply:
Are they really test-only? I think the integration tests need to work with JSON.

Reply:
At least for this module they are in tests.

Reply:
Similar issue mentioned here: https://stackoverflow.com/questions/77360885/maven-dependency-plugin-3-6-started-to-find-new-used-undeclared-dependencies

I am re-thinking, is it worth fixing this somehow or should we wait until we get a suitable version of this plugin and do the upgrade then. I am not sure if we have to upgrade because according to [this](https://mvnrepository.com/artifact/org.apache.maven.plugins/maven-dependency-plugin), there are no vulnerabilities in the current versions (3.1.2 -> root, 3.3.0 -> flight-core).  

What do you think @lidavidm?

Reply:
We can just ignore this version for now then

Reply:
Sure. I will close this one.